CHANGELOG.md

```diff
@@ -13,6 +13,10 @@
 
 - Reduced `fps_simulated_players_creation_interval` from 60 to 5.
 - Disabled the `Generate Map` button in the MapBuilder window if `MapBuilderSettings` is not set.
+- The four middle tiles of the world are no longer forced to be of certain tile type.
+- World tile prefabs improved.
+	- Now make use of nested prefabs.
+	- Completely new set of default tiles used to populate the world
```

Comment:
once the other map builder PR goes in, please update the changelog

Reply:
👍 👍